sharqiyya app/header.css

```diff
@@ -0,0 +1,84 @@
+.header{
+  align-self: flex-start;
+  display: flex;
+  flex-direction: column;
+  font-family: Montserrat;
+  width: 100%;
+  height: 118px;
+}
+
+.app-bar{
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+  width: 100%;
```

Comment:
The app width should not expand after a certain width. On very wide screens the the content should be have a max-width. The background color fo the header and footer are the only things that expand with the screen.